src/components/Alerts/src/stories/Alerts.stories.mdx

```diff
@@ -0,0 +1,159 @@
+import { ArgsTable, Canvas, Description, Meta, Story } from "@storybook/addon-docs/blocks";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import { action } from "@storybook/addon-actions";
+import { Alerts } from "..";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="Components/Data Display/Alerts"
+  component={Alerts}
+  parameters={{ componentSubtitle: `${pkg.name} - ${pkg.version}`, docs: { source: { type: "dynamic" } } }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Alerts" viewType="table" />
+
+
+## Stories
+
+<ArgsTable of={Alerts} />
+
+### Error
+
+<Canvas>
+  <Story
+    name="Error"
+    args={{
+      title: "Title",
+      text: "Here comes text. This text provides additional details and actionable steps the user can take.",
+      variant: "error",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+import { Alert } from "@gemeente-denhaag/alert";
+const ErrorAlert = <Alerts
+  text="Here comes text. This text provides additional details and actionable steps the user can take."
+  title="Title"
+  variant="error"
+/>
+// This 👆 transpiles into this 👇 in the DOM
+<div class="denhaag-alert" variant="error">
+  <div class="denhaag-alert__icon">
+    <svg width="24" height="24" ...>...</svg>
+  </div>
+  <div class="denhaag-alert__content">
+    <h4 class="utrecht-heading-4 utrecht-heading-4--distanced">Title</h4>
+    <p class="utrecht-paragraph utrecht-paragraph--distanced">
+      Here comes text. This text provides additional details and actionable steps the user can take.
+    </p>
+  </div>
+</div>
```

Comment:
This HTML snippet shows only the most basic alert. If I was an external user and wanted to copy our BEM structure, but use the action and close button as well, I'd be at a loss for where to put it unless I dug into the src on github.

Reply:
Agreed, I've added a second html snippet to the example with both action and close buttons.
There's one downside though, the close button is an Iconbutton which still needs MUI classes, so those are also added to the snippet..